app/models/user.rb

```diff
@@ -1,10 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts, foreign_key: 'author_id'
   has_many :comments
   has_many :likes
 
   validates :name, length: { minimum: 1 }
```

Comment:
Hello @Kingstalux Are you sure that's was the correct branch you pulled?...because I have every thing set up here
![CaptureXIII](https://user-images.githubusercontent.com/83566118/173555181-f278af98-1c27-4f00-9fa3-561fbd3b193d.PNG)

Reply:
Hello @oluyaratosin123, try dropping your database and then creating a new user.

Reply:
> Hello @oluyaratosin123, try dropping your database and then creating a new user. A new user is created on the signup page and see if that works. Also, have you set up devise in any other branch?

Just this branch alone